zstash/extract.py

```diff
@@ -365,16 +369,23 @@ def extractFiles(  # noqa: C901
 
         try:
             # Seek file position
-            # FIXME: Item "None" of "Optional[IO[bytes]]" has no attribute "seek" mypy(error)
-            tar.fileobj.seek(file_tuple[6])  # type: ignore
+            if tar.fileobj:
+                fileobj = tar.fileobj
+            else:
+                raise Exception("Invalid tar.fileobj={}".format(tar.fileobj))
+            fileobj.seek(file_tuple[6])
```

Comment:
Does this work instead of declaring a `fileobj` var?

```
if tar.fileobj:
    fileobj.seek(file_tuple[6])
else:
    raise Exception("Invalid tar.fileobj={}".format(tar.fileobj))
```